.github/workflows/ci.yaml

```diff
@@ -221,6 +221,12 @@ jobs:
       - name: Add SBT proxy repositories
         run: cp -vf .github/workflows/repositories /root/.sbt/ ; true
 
+      - name: Disable experimental features on release
+        if: "github.event_name == 'push'
+            && startsWith(github.event.ref, 'refs/tags/')
+            && !startsWith(github.event.ref, 'refs/tags/sbt-dotty-')"
```

Comment:
We probably don't need the check for an `sbt-dotty` release tag here (and below) since `sbt-dotty` has been removed on master.